plugins/hls-pragmas-plugin/test/Main.hs

```diff
@@ -114,6 +114,11 @@ codeActionTests' =
       cas <- map fromAction <$> getAllCodeActions doc
       liftIO $ "Disable \"deferred-type-errors\" warnings" `notElem` map (^. L.title) cas @? "Doesn't contain deferred-type-errors code action"
       liftIO $ length cas == 0 @? "Expected no code actions, but got: " <> show cas
+  , goldenWithPragmas pragmasSuggestPlugin "doesn't suggest disabling out of scope variables" "DeferredOutOfScopeVariables" $ \doc -> do
```

Comment:
```suggestion
  , goldenWithPragmas pragmasDisableWarningPlugin "doesn't suggest disabling out of scope variables" "DeferredOutOfScopeVariables" $ \doc -> do
```

Wrong plugin :) Same goes for the other test, if you dont mind, could you update that test as well? 

Thanks!

Reply:
Yep, that was it. Both test now actually work.